### PR TITLE
Create Hrmn97.json

### PR DESCRIPTION
**Pull request by Hrmn97**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4032"></a>

### CODE REVIEW

**Review by AbbeyIT — CHANGES_REQUESTED** [review]
Hi kindly rename your file accordingly with your GitHub username which is `Hrmn97`.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Sindhuinti — APPROVED** [review]

File: data/Hrmn97.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "url": "https://twitter.com/Harman9765",
+        "icon": "FaTwitter"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/h_97singh/",
+        "icon": "FaInstagram"
+      },
+      {
+        "name": "COnnect with me on GitHub",
```
Comment (SaraJaoude) [inline data/Hrmn97.json:24]: Not a blocker but you may want to fix this in a future PR

```suggestion
        "name": "Connect with me on GitHub",
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**